config/connection.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * @Author : Mfsi_Annapurnaa
+ * @purpose : Db connection operations
+ */
+
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
+
+// Include constants
+include('config/constants.php');
+
+class connection
+{
+
+    static $conn;
```

Comment:
Should not we specify scope of this variable. by default its public but i think better to specify?